celements-model/src/main/java/com/celements/auth/user/UserService.java

```diff
@@ -150,12 +150,32 @@ boolean addUserToGroup(@NotNull User user, @NotNull ClassReference groupRef)
    * @param possibleLoginFields
    *          list of user fields for which one can login, e.g. 'name', 'email' or 'validkey'
    * @return the matched {@link User} instance or absent if the given login doesn't match or isn't
-   *         unique
+   *         unique. Excludes suspended Users from the return value.
    */
+  /**
+   * @deprecated instead use {@link #getPossibleUserForLoginField} with
+   *             {@link User#isSuspended()}
+   * @since 6.5
+   */
+  @Deprecated(since = "6.5", forRemoval = true)
   @NotNull
   Optional<User> getUserForLoginField(@NotNull String login,
       @Nullable Collection<String> possibleLoginFields);
 
+  /**
+   * looks up the user by the given login value with the possible login fields.
+   *
+   * @param login
+   *          the login value
+   * @param possibleLoginFields
+   *          list of user fields for which one can login, e.g. 'name', 'email' or 'validkey'
+   * @return the matched {@link User} instance or absent if the given login doesn't match or isn't
+   *         unique. includes suspended Users in return value.
+   */
+  @NotNull
+  java.util.Optional<User> getPossibleUserForLoginField(@NotNull String login,
```

Comment:
import `java.util.Optional` and instead fully qualify the guava optional to make it obvious those need refactoring/deprecation